src/Traits/AttributeResolverTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * Parser Reflection API
+ *
+ * @copyright Copyright 2015, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\ParserReflection\Traits;
+
+use Go\ParserReflection\ReflectionAttribute;
+
+trait AttributeResolverTrait
+{
+    /**
+     * @return ReflectionAttribute[]
+     */
+    public function getAttributes(?string $name = null, int $flags = 0): array
+    {
+        $this->__initialize();
```

Comment:
Initialization is not allowed here, as it will trigger the load of original class and for the `goaop/framework` this won't work, as we will be already inside class loading process, so fatal error will occur that class has not been loaded yet.

So, this call should be disabled and all attributes should be crafted from the corresponding AST nodes here. 

To have an idea - it might be good to have a look at `\Go\ParserReflection\ReflectionMethod::collectFromClassNode` and create something similar, eg `\Go\ParserReflection\ReflectionAttribute::collectFromAttrGroup(AttrGroup $attrGroup, ReflectionClass|ReflectionMethod|ReflectionParameter|<anything else containing attribute> $reflector): ReflectionAttribute[]

Reply:
`__initialize()` were not always executed before, that's why I added the new method `__initialize()` in some reflections in this PR, otherwise we got fatal error on `getAttributes()` call.

 I will check if init can be called early in the reflection itself.